ceph/rbd/workflows/rbd.py

```diff
@@ -45,82 +218,59 @@ def config_rbd_multi_pool(
     client,
     is_secondary=False,
     cluster="ceph",
+    create_pool_parallely=False,
+    create_image_parallely=False,
 ):
-    """ """
+    """
+    Create number of pools as specified in parallel/sequential
```

Comment:
return and args are missing, please add these in all methods along with examples config.

Reply:
config example? i didn't understand what you mean by that?

Reply:
The input parameters are all self explanatory. We don't need to explain it I feel.